packages/smart-contract/src/contracts/zkdb-app.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  Field,
+  MerkleTree,
+  PublicKey,
+  Reducer,
+  SmartContract,
+  State,
+  ZkProgram,
+  method,
+  state,
+} from 'o1js';
+import { RollUpProgram } from '../proof/proof-program';
+import { Action } from '../archive-node/action';
+
+export function getZkDbSmartContract(
```

Comment:
Can we implement `ZKDatabaseContract` as a class then allowed other class to be extended from `ZKDatabaseContract`.

Reply:
This function returns the class, so we can extend from the function. I think, I will come up with the better when dealing with sdk